magenta/models/music_vae/js/dist/musicvae.js

```diff
@@ -387,8 +401,18 @@ var Decoder = (function () {
 exports.Decoder = Decoder;
 var MusicVAE = (function () {
     function MusicVAE(checkpointURL, dataConverter) {
+        var _this = this;
         this.checkpointURL = checkpointURL;
-        this.dataConverter = dataConverter;
+        if (dataConverter) {
+            this.dataConverter = dataConverter;
+        }
+        else {
+            fetch(checkpointURL + '/converter.json')
+                .then(function (response) { return response.json(); })
+                .then(function (converterSpec) {
+                _this.dataConverter = data.converterFromSpec(converterSpec);
```

Comment:
Rather than _this, I think you can use .bind: https://stackoverflow.com/questions/16435167/binding-a-promise-handler-function-to-an-object

Reply:
You can also use an arrow function, which leaves `this` alone. (Unless you're avoiding arrows in this project.)

Reply:
this is the bundle